Cargo.toml

```diff
@@ -61,11 +61,7 @@ members = [
 ]
 exclude = [
     # Examples for how to set up an independent Cargo.toml file
-    "docs/tutorials/cargo_tests/baked",
-    "docs/tutorials/cargo_tests/buffer",
-    "docs/tutorials/cargo_tests/experimental",
-    "docs/tutorials/cargo_tests/lazy_static",
-    "docs/tutorials/cargo_tests/testdata",
+    "docs/tutorials/cargo_tests",
```

Comment:
I think other workspaces are automatically excluded. We don't exclude the fuzzing crates or `icu_capi_staticlib_tiny` that declare they're a workspace.

Reply:
Hmm, maybe. I don't want to get messy with global feature resolution so I'll add a comment saying this is maybe not necessary

Reply:
Let's try to be consistent and remove this or add all non-workspace crates

Reply:
ok